sw/tools/iridium/iridium_link.py

```diff
@@ -0,0 +1,220 @@
+    self.lbl_number = gtk.Label("Phone number")
+    self.status_wind.add(self.lbl_number)
+    self.entry_number = gtk.Entry()
+    self.entry_number.set_text(DEFAULT_NUMBER)
+    self.status_wind.add(self.entry_number)
+
+    # Setting - Redail
+    self.lbl_redail = gtk.CheckButton("Auto redail")
+    self.status_wind.add(self.lbl_redail)
+
+    # Status
+    self.status_wind.add(gtk.HSeparator())
+    self.lbl_connected = gtk.CheckButton("Connected")
+    self.lbl_connected.set_sensitive(False)
+    self.status_wind.add(self.lbl_connected)
+    self.lbl_configured = gtk.CheckButton("Configured")
+    self.lbl_configured.set_sensitive(False)
+    self.status_wind.add(self.lbl_configured)
+    self.lbl_registered = gtk.CheckButton("Registered")
+    self.lbl_registered.set_sensitive(False)
+    self.status_wind.add(self.lbl_registered)
+    self.lbl_in_call = gtk.CheckButton("In call")
+    self.lbl_in_call.set_sensitive(False)
+    self.status_wind.add(self.lbl_in_call)
+
+    # Add a console output
+    self.sw_console = gtk.ScrolledWindow()
+    self.sw_console.set_size_request(700,400)
+    self.console = gtk.TextView()
+    self.console.set_editable(False)
+    self.ir.set_console(self.console_write)
+    self.sw_console.add(self.console)
+    self.hbox.add(self.sw_console)
+
+    # Create a button toolbar
+    self.toolbar = gtk.HBox()
+    self.toolbar.set_size_request(0,40)
+    self.vbox.pack_start(self.toolbar, expand=False)
+
+    # Create the open device
+    self.btn_open = gtk.Button("Open")
+    self.btn_open.connect("clicked", self.open, None)
+    self.toolbar.add(self.btn_open)
+
+    # Create the reguster button
+    self.btn_register = gtk.Button("Register")
+    self.btn_register.connect("clicked", self.register, None)
+    self.btn_register.set_sensitive(False)
+    self.toolbar.add(self.btn_register)
+
+    # Create the call button
+    self.btn_call = gtk.Button("Call")
+    self.btn_call.connect("clicked", self.call, None)
+    self.btn_call.set_sensitive(False)
+    self.toolbar.add(self.btn_call)
+
+    # Create the CSQ button
+    self.btn_csq = gtk.Button("CSQ")
+    self.btn_csq.connect("clicked", self.csq, None)
+    self.btn_csq.set_sensitive(False)
+    self.toolbar.add(self.btn_csq)
+
+    self.window.set_icon_from_file(os.getenv("PAPARAZZI_HOME") + "/sw/tools/iridium/sat.ico")
+
+    # Show the window everything
+    self.window.show_all()
+
+  def destroy(self, widget, data=None):
+    reactor.stop()
+
+class PprzUDPProtocol(DatagramProtocol):
+  cb_recv_data = False
+
+  def __init__(self, in_port, out_port, cb_recv_data):
+    self.port = in_port
+    self.out_port = out_port
+    self.cb_recv_data = cb_recv_data
+
+  def startProtocol(self):
+    self.transport.socket.setsockopt(SOL_SOCKET, SO_BROADCAST, True)
+
+  def sendData(self, data):
+    self.transport.write(data, ('127.255.255.255', self.out_port))
+
+  def datagramReceived(self, datagram, addr):
+    if self.cb_recv_data != None:
```

Comment:
Please not that checking for None should not be done with comparison operators, but with `is` and `is not`.
See https://www.python.org/dev/peps/pep-0008/#programming-recommendations
http://stackoverflow.com/questions/14247373/python-none-comparison-should-i-use-is-or